src/components/ContactGroups/ContactGroupManager.spec.jsx

```diff
@@ -1,32 +1,67 @@
 import React from 'react'
-import renderer from 'react-test-renderer'
+import { render, fireEvent } from '@testing-library/react'
+
+import { groups } from '../../helpers/testData'
 import ContactGroupManager from './ContactGroupManager'
 import AppLike from '../../tests/Applike'
-import { DOCTYPE_CONTACT_GROUPS } from '../../helpers/doctypes'
+
+const createGroup = jest.fn()
+const renameGroup = jest.fn()
+
+const jsx = (
+  <AppLike>
+    <ContactGroupManager
+      contactGroups={[groups[0]]}
+      allGroups={groups}
+      onGroupSelectionChange={() => {}}
+      createGroup={createGroup}
+      deleteGroup={() => {}}
+      renameGroup={renameGroup}
+    />
+  </AppLike>
+)
```

Comment:
Nit: We have taken the habit of using a `setup` function to share code between tests.

```
const setup = () => {
	const root = render(
	  <AppLike>
	    <ContactGroupManager
	      contactGroups={[groups[0]]}
	      allGroups={groups}
	      onGroupSelectionChange={() => {}}
	      createGroup={createGroup}
	      deleteGroup={() => {}}
	      renameGroup={renameGroup}
	    />
	  </AppLike>
	)
	return { root }
}

it('should....', () => {
  const { root } = setup()
})
```

I think we could use this instead of the jsx variable, it'd be a bit more coherent with the rest of the cozy codebase.

The setup function can accept options to achieve different scenarios.

```
setup({ allGroups: [] }) // Test with no groups for example
```

Reply:
Top, je corrige !